### DISCUSSION TITLE
Adonis JS Geometry Datatype

### DISCUSSION THREAD

**Discussion by Areebsofyrus** [adonisjs/core#2437] [Help]
I cant find any column type in schema  builder of AdonisJS for dealing with geospatial datatype as it is present as "geometry" datatype in other schemabuilders such as Laravel Schema builder or SEQUALIZE schema.

Please help me out that how can i deal with geospatial/locations data in Adonis Application ?

**Comment by thetutlage** [comment]
There is no inbuilt method for it yet. However, you can define it using the `specificType` method.

```ts
table.specificType('locations', 'MULTIPOINT').notNullable()
```

**Reply by Areebsofyrus** [reply]
Thank you @thetutlage,will try it
